rotkehlchen/exchanges/coinbase.py

```diff
@@ -66,10 +66,11 @@ def trade_from_coinbase(raw_trade: Dict[str, Any]) -> Optional[Trade]:
         # We only want to deal with completed trades
         return None
 
-    if raw_trade['instant']:
-        raw_time = raw_trade['created_at']
-    else:
-        raw_time = raw_trade['payout_at']
+    # Contrary to the Coinbase documentation we will use created_at, and never
+    # payout_at. It seems like payout_at is not actually the time where the trade is settled.
+    # Reports generated by Coinbase use created_at as well
+    raw_time = raw_trade['created_at']
```

Comment:
what if created_at does not exist? Is that possible? Maybe in that case use payout_at as a backup?